foundations/installations/prerequisites.md

```diff
@@ -188,6 +188,18 @@ Installing Ubuntu is where the real changes start happening on your computer. Th
 
 For step-by-step instructions, please follow this [installation guide](https://tutorials.ubuntu.com/tutorial/tutorial-install-ubuntu-desktop#0) from the creators of Ubuntu.
 
+#### Step 3.3: Configure BIOS/UEFI settings
+
+Now you are going to set up your boot settings so that when you boot your pc it will open up Ubuntu straight away and also give you the option to boot with Windows. 
+
```

Comment:
This needs to include a sentence about the fact that the exact steps will vary between computers.